frontend/src/components/dashboard/subdomain/SubdomainInfoTooltip.module.scss

```diff
@@ -0,0 +1,73 @@
+@import "../../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "~@mozilla-protocol/core/protocol/css/includes/mixins/typography";
+
+.info-icon {
+  align-items: center;
+  background-color: transparent;
+  border-style: none;
+  color: $color-light-gray-70;
+  cursor: pointer;
+  display: flex;
+}
+
+.explainer-wrapper {
+  background-color: $color-white;
+  box-shadow: $box-shadow-sm;
+  border-radius: $border-radius-md;
+  padding: $spacing-lg;
+  // On small screens, appear as a modal at the top of the viewport:
+  top: $spacing-lg;
+  left: $spacing-lg;
+  right: $spacing-lg;
+  position: absolute;
+
+  &::before {
+    content: "";
+    height: 12px;
+    width: 12px;
+    position: absolute;
+    top: -6px;
+    left: 12px;
+    transform: rotate(45deg);
+    background-color: $color-white;
+    display: none;
+  }
+
+  @media screen and (min-width: 970px) {
+    // On wider screens, the popover is attached to the indicator:
+    inset: unset;
+    position: unset;
+    max-width: 546px;
```

Comment:
suggestion: use content tokens if possible

https://protocol.mozilla.org/docs/fundamentals/design-tokens.html

`$content-sm`, etc.

Reply:
Should I prioritize using tokens, or matching the width in the figma file? The width in the figma file was 546 px, and the tokens were a little far off from this.

Reply:
From our end, I recommend using your own judgement. There's a "transition" from Figma to our system, where we get as close as we can to the design, using tokens. For the most part, the design team is really good about using tokenized colors, etc. It's rare to go use a static number. 

The main thing is to have a conversation with the designer about any changes you make to confirm it's okay. 9/10 times, it's totally fine!